common/lib/xmodule/xmodule/conditional_module.py

```diff
@@ -160,8 +160,9 @@ def is_condition_satisfied(self):
                     # the descriptor of a required module to have a property but
                     # for the resulting module to be a (flavor of) ErrorModule.
                     # So just log and return false.
-                    log.warn('Error in conditional module: \
-                        required module {module} has no {module_attr}'.format(module=module, module_attr=attr_name))
+                    if module is not None:
```

Comment:
[`get_module`](https://github.com/edx/edx-platform/blob/master/common/lib/xmodule/xmodule/conditional_module.py#L150) has been used in [`required_modules`](https://github.com/edx/edx-platform/blob/master/common/lib/xmodule/xmodule/conditional_module.py#L148-L151) which may return [`None`](https://github.com/edx/edx-platform/blob/master/lms/djangoapps/courseware/module_render.py#L446) if requester does not have access to that module – [see more](https://github.com/edx/edx-platform/blob/master/lms/djangoapps/courseware/module_render.py#L763-L765).